pkg/operator/staticpod/certsyncpod/certsync_cmd.go

```diff
@@ -55,6 +66,8 @@ func NewCertSyncControllerCommand(configmaps, secrets []installer.UnrevisionedRe
 	cmd.Flags().StringVar(&o.KubeConfigFile, "kubeconfig", o.KubeConfigFile, "Location of the master configuration file to run from.")
```

Comment:
Can I encapsulate these in a new options struct?

Reply:
What for?

Reply:
for separation of concern and using similar approach to other places